README.md

```diff
@@ -1233,6 +1234,36 @@ datetime                      value
 2019-09-03 00:00:00+00:00    9.0
 ```
 
+### Trending Words
```

Comment:
Let's use the new name `Emerging Trends` instead of `Trending Words` even though the API has the old name